client/src/helpers/APICalls/imageUpload.ts

```diff
@@ -0,0 +1,34 @@
+// import { FetchOptions } from '../../interface/FetchOptions';
+
+// body is not string in this request
+interface FetchOptions {
+  method: string;
+  headers?: {
+    'Content-Type': string;
+  };
+  body?: any;
+  credentials: RequestCredentials;
+}
+
+const uploadImage = async (images: any): Promise<any> => {
+  const fd = new FormData();
+
+  for (const image of images) {
+    fd.append('images', image, `${image.name}`);
+  }
+
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    // headers: { 'Content-Type': 'multipart/form-data; boundary=AaB03x' },
```

Comment:
I would add headers in here

Reply:
commenting that out was the only way I could get it working, any ideas why?

Reply:
Could you provide some further context into what is happening when that header is set?

Reply:
the image doesn't upload, and on the server-side the body contains nothing. other than that the endpoint returns a Success status.